src/knx/table_object.h

```diff
@@ -3,6 +3,9 @@
 #include "interface_object.h"
 
 class Memory;
+
+typedef void (*beforeTablesUnloadCallback)();
```

Comment:
Same as above: please start the type with a capital letter.

Reply:
Done